recipes/libdwarf/all/conanfile.py

```diff
@@ -50,6 +53,8 @@ def requirements(self):
         if self.options.with_dwarfgen or self.version == "20191104":
             self.requires("libelf/0.8.13")
         self.requires("zlib/[>=1.2.11 <2]")
+        if self.version != "20191104" and Version(self.version) >= Version("0.9.0"):
+            self.requires("zstd/[>=1.3.5]")
```

Comment:
```suggestion
            self.requires("zstd/1.5.5")
```

Please, do not use version ranges. It's allowed only to few packages: https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges